### PR TITLE
ADD doc test plan

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers
@flopezag @hmunfru 

#### Description
UPDATE Test project documentation and Test Plan for release 4.1

### CODE REVIEW

**Comment by hmunfru** [comment]
LGTM

File: test/acceptance/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+===========================
+PAAS API - Acceptance Tests
+===========================
+This project contains the PAAS acceptance tests (component, integration and E2E testing).
```
Comment (flopezag) [inline test/acceptance/README.rst]: Change PAAS by Pegasus (PaaS)

File: test/acceptance/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Test environment
+----------------
+
+**Prerequisites**
+
+- Python 2.7 or newer (2.x) (https://www.python.org/downloads/)
+- pip (https://pypi.python.org/pypi/pip)
+- virtualenv (https://pypi.python.org/pypi/virtualenv)
+- PAAS (`Download PaaS <http://catalogue.fi-ware.org/enablers/paas-manager-pegasus/downloads>`_)
```
Comment (flopezag) [inline test/acceptance/README.rst]: PAAS -> Pegasus

Download Pegasus (PaaS Manager) ....

File: test/acceptance/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+1. Create a virtual environment somewhere *(virtualenv ~/venv)*
+#. Activate the virtual environment *(source ~/venv/bin/activate)*
+#. Go to *test/acceptance* folder in the project
+#. Install the requirements for the acceptance tests in the virtual environment *(pip install -r requirements.txt --allow-all-external)*
+
+
+Test structure and prerequisites
+---------------------------------
+
+PaaS will need to be deployed in a Fiware environment to be tested.
```
Comment (flopezag) [inline test/acceptance/README.rst]: Pegasus (PaaS Manager)

File: test/acceptance/README.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Acceptance tests have two type of test cases:
+
+- **Catalog test cases**: They try to test PaaS catalog and they needn't additional prerequisites.
```
Comment (flopezag) [inline test/acceptance/README.rst]: Pegasus

they do not need

File: test/acceptance/README.rst [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+- **Instance test cases**: Those test cases try to test PaaS Blueprint deployment.
```
Comment (flopezag) [inline test/acceptance/README.rst]: Pegasus

File: test/acceptance/README.rst [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+**NOTE**: For launching blueprints you will need the full OpenStack architecture available, the product releases 
+registered in SDC and Chef-Server/Puppet-Master must be ready to install the software requested.
```
Comment (flopezag) [inline test/acceptance/README.rst]: SDC -> Sagitta 

Please add reference to the catalog about this component.

File: test/acceptance/README.rst [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Configuration file
+------------------
+Some configuration is needed before test case execution. This configuration is set in the *properties.json* file.
```
Comment (flopezag) [inline test/acceptance/README.rst]: You should add details about how to introduce the data about the user, tenant, password, ... Not explicitly introduce in github but you need to explain how to obtain them or how to fix them into the properties.json

File: test/acceptance/README.rst [hunk 8]
```diff
@@ -0,0 +1,78 @@
+===========================
+PAAS API - Acceptance Tests
```
Comment (flopezag) [inline test/acceptance/README.rst]: Pegasus API

File: test/README.rst [hunk 9]
```diff
@@ -0,0 +1,8 @@
+Folder for automated functional tests of the SDC API
```
Comment (flopezag) [inline test/README.rst]: SDC -> Sagitta

File: test/doc/roadmap_unit_testing_plan.rst [hunk 10]
```diff
@@ -0,0 +1,130 @@
+==============
+PaaS Test Plan
```
Comment (flopezag) [inline test/doc/roadmap_unit_testing_plan.rst]: Pegasus Test Plan

File: test/doc/roadmap_unit_testing_plan.rst [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+	17 scenarios (17 passed)
+
+- create_environment_with_tiers.feature ::
+	
+	1 feature (1 passed)
+	18 scenarios (18 passed)
+	
+**Provisioning Test Cases**
+
+You can execute E2E test cases for this release executing this command: ::
```
Comment (flopezag) [inline test/doc/roadmap_unit_testing_plan.rst]: it is correct to put several colons...

File: test/doc/roadmap_unit_testing_plan.rst [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+	3 scenarios (3 passed)
+----------------------------
+
+**FIWARE.Feature.Cloud.PaaSManager.GEIdentification** ::
+
+   The VM should contain a metadata section with the GE Id. This is needed for some
+   GEs in order to manage the GEs deployed in the Cloud. The GE identification 
+   is called NID. 
+
+Test Cases involved in this feature should validate that PaaS Manager is creating a NID metadata in each VM when blueprint is deployed.
```
Comment (flopezag) [inline test/doc/roadmap_unit_testing_plan.rst]: Pegasus (PaaS Manager)

File: test/doc/roadmap_unit_testing_plan.rst [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+
+
+**FIWARE.Feature.Cloud.PaaSManager.ConfigurationManagement** ::
+
+	Complex Blueprint templates have software attributes dependences. 
+	PaaS Manager should be able to resolve this complex configuration and translate it
+	to recipes. Concretely, it means to resolve macros introduced by the user 
+	or the portal. Some example can be IP VM, or a software attribute. 
+   
+Test Cases involved in this feature should validate that PaaS Manager is managing IP translation regarding type of the product attribute (Plain, IP, IPALL) and the name of the tier.
```
Comment (flopezag) [inline test/doc/roadmap_unit_testing_plan.rst]: Idem to the previous.

File: test/doc/roadmap_unit_testing_plan.rst (hunk 11 above)
Comment (jframos) [inline test/doc/roadmap_unit_testing_plan.rst]: Following reStructuredText markup syntax, I can use double colons to represent literal blocks.

Comment (flopezag) [inline test/doc/roadmap_unit_testing_plan.rst]: NTC

File: test/acceptance/README.rst (hunk 7 above)
Comment (jframos) [inline test/acceptance/README.rst]: Added below the json example.

**Comment by jframos** [comment]
ping @flopezag

**Comment by flopezag** [comment]
LGTM

**Comment by flopezag** [comment]
I suggest to close your own branch...

**Pull request merged**